kore/src/Kore/Log/KoreLogOptions.hs

```diff
@@ -201,12 +205,15 @@ parseEntryTypes =
             [ "Log entries: logs entries of supplied types"
             , "Available entry types:"
             , (OptPretty.indent 4 . OptPretty.vsep)
-                (OptPretty.text <$> getEntryTypesAsText)
+                ( OptPretty.text
+                    <$> getNoErrEntryTypesAsText <> getErrEntryTypesAsText
```

Comment:
I think this could remain `getEntryTypesAsText`, and that function could be implemented in `Registry`.